app/controllers/concerns/set_for_end_of_month.rb

```diff
@@ -1,11 +1,14 @@
 module SetForEndOfMonth
   def set_for_end_of_month
+    # Defaults to current month if it's the last week of the month.
+    @show_current_month = ((Date.current.end_of_month - Date.current).to_i <= 7)
+
     @for_end_of_month ||= if params[:for_end_of_month]
```

Comment:
Calling this with a `for` prefix seems weird. How about just `end_of_month`? Or `end_of_latest_month`?